content/concepts/multiplex/mplex.md

```diff
@@ -1,14 +1,48 @@
 ---
 title: "mplex"
-description: "mplex is a protocol developed for libp2p."
+description: "mplex is a stream multiplexer developed for libp2p."
 weight: 170
 ---
 
-mplex is a protocol developed for libp2p. The [spec](https://github.com/libp2p/specs/tree/master/mplex) defines a
-simple protocol for multiplexing that is widely supported across libp2p language implementations:
+## What is mplex?
 
-{{< alert icon="💡" context="note">}}
-This section is incomplete, and many of the articles are stubs. To help fill in
-the gaps, please see the issues linked in each article to add your input and
-help us prioritize the outstanding work.
+mplex is a stream multiplexer designed for libp2p.
+It is a simple protocol that does not provide many features offered by other
+stream multiplexers. Notably, mplex does not provide backpressure at the protocol
+level.
+
+mplex runs over a reliable, ordered pipe between two peers, such as a TCP socket
+or a Unix pipe. Peers can open, write to, close, and reset a stream by sending messages
+with the appropriate [flag value](https://github.com/libp2p/specs/tree/master/mplex#flag-values).
+
+Every message in mplex consists of a header and a length of prefixed data
+segment.
+
+### Drawbacks
+
+mplex does not support backpressure because it is designed to be more flexible and
+extensible than other multiplexing protocols. Instead of using a flow control mechanism
```

Comment:
It's not really flexible and extensible. I think it was primarily designed for implementation simplicity, without really considering that backpressure is kind of essential when writing a networking stack. Of course we shouldn't put it like this in our docs ;)